app/Http/Controllers/ReservationController.php

```diff
@@ -111,32 +112,70 @@ public function index()
     public function store(Request $request)
     {
 
-        //   $request->validate([
-        //      'name' => 'required'
-        // ]);
+        $request->validate([
+            'name' => 'required',
+            'email' => 'required',
+            'tel' => 'required',
+            'consent' => 'required'
+        ]);
+        $seatsData =  json_decode($request->input('seats'), true);
+
+        if ($request->input('stand') == 0 &&  count($seatsData ?? []) == 0) {
```

Comment:
is the double space after double && okay? the same after` $seatsData = `

Reply:
just formatting issue, no impact on functionality